version.mk

```
@@ -0,0 +1,5 @@
+COMMIT_NUMBER=$(shell git rev-list `git rev-list --parents HEAD | egrep "^[a-f0-9]{40}$$"`..HEAD --count)
+BUILD_DATE=$(shell date --utc +%Y-%m-%d)
```

```suggestion
BUILD_DATE=$(shell date -u +%Y-%m-%d)
```
`--utc` is not portable:

    lseelye@molydbenum namespace $ uname -a
    Darwin molydbenum 18.2.0 Darwin Kernel Version 18.2.0: Thu Dec 20 20:46:53 PST 2018; root:xnu-4903.241.1~1/RELEASE_X86_64 x86_64
    lseelye@molydbenum namespace $ date --utc +%Y-%m-%d
    date: illegal option -- -
    usage: date [-jnRu] [-d dst] [-r seconds] [-t west] [-v[+|-]val[ymwdHMS]] ... 
                [-f fmt date | [[[mm]dd]HH]MM[[cc]yy][.ss]] [+format]